contracts/ProtocolWallet.sol

```diff
@@ -3,8 +3,10 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "./spec_interfaces/IProtocolWallet.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
 import "./ContractRegistryAccessor.sol";
+import "./Lockable.sol";
```

Comment:
The protocol wallet is not lockable, not managed and should not have access to ContractRegistryAccessor.